app.js

```diff
@@ -277,10 +278,12 @@ app.get('/api/home/screenings', async (request, response) => {
   const screenings = await homeScreening(getTenScreeningsAdapter);
   response.json(screenings);
 });
+
 // API route for individual movie page screenings (client-side fetching)
 app.get('/movie/:id/screenings', async (request, response) => {
   const movieId = request.params.id;
-  moviePage(response, `/api/screenings?filters[movie]=${movieId}`);
+  const queryString = `?filters[movie]=${movieId}`;
+  moviePage(response, cmsAdapter, queryString);
```

Comment:
I would accept a return value here, and then do `response.json(theReturnValue)`. That way, the `moviePage` function doesn't have to also understand Response objects, thereby reducing it's concerns (for better separation of concerns).